### ISSUE TITLE
how to download cityscapes demo videos?

### ISSUE THREAD

**Issue by haochenheheda** [ansleliu/LightNet#9]

**Comment by haochenheheda** [comment]
@ansleliu 
sorry , i mean this video without mask,  ` video_path = "/zfs/zhang/Cityscapes/leftImg8bit/demoVideo"`
  
 and another question, Is the model trained from scratch?    I see two argument,--resume and --pre-trained in the code, but no pkl in this project, is it pretrained on another datasets?
thank you for your answer!

**Comment by ansleliu** [comment]
@haochenheheda 
1. You can download demo video without masks from the [official website of cityscapes](https://www.cityscapes-dataset.com/downloads/).
2. The base-net (original mobilenetv2) was pre-trained using ImageNet, and LightNets (mobilenetv2plus, rfmobilenetplus etc.) are fit using Cityscapes "gtFine" & "GAN for data augmentation". ^_^

**Comment by haochenheheda** [comment]
@ansleliu Thank you very much for your answer！  
Sorry to bother you again，and this is the last time ,I promise ~  
I train from scratch and the result is not so good. I will be very grateful if you can release your pretrained weight,  My coding ability is very poor and I don't want to pretrain by myself ≥＾≤.hh

**Comment by ansleliu** [comment]
@haochenheheda You can find the download link for the checkpoint in the README. ^_^
